native-server/src/main/java/com/datastax/oss/simulacron/server/BoundNode.java

```diff
@@ -423,6 +423,7 @@ void handle(ChannelHandlerContext ctx, Frame frame) {
             response = new SetKeyspace(keyspace);
           }
         } else {
+          logger.warn("No stub mapping found for message type QUERY: \"{}\"", queryStr);
```

Comment:
How about adding a DEBUG-level message on line 365 to tell users that we did find a stub mapping? That could also be helpful when debugging Simulacron imo.

Reply:
yeah, good idea. Will add it in.

Reply:
added new debug log line in latest commit